cli/lib/kontena/cli/stacks/show_command.rb

```diff
@@ -32,6 +53,10 @@ def show_stack(name)
       puts "  version: #{stack['version']}"
       puts "  revision: #{stack['revision']}"
       puts "  expose: #{stack['expose'] || '-'}"
+      puts "  variables:#{' -' if stack['variables'].nil? || stack['variables'].empty?}"
+      stack['variables'].each do |var, val|
```

Comment:
The "stacks" created by the `kontena registry`/`vpn` commands unfortunately seem to be return `{"variables": null}` from the API... that should maybe be fixed somehow, as it's breaking this `kontena stack show registry`:

```
NoMethodError: undefined method `each' for nil:NilClass
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/show_command.rb:57:in `show_stack'
  /home/kontena/kontena/kontena/cli/lib/kontena/cli/stacks/show_command.rb:21:in `execute'
```